src/Test/Utilities/Portable/CompilerTraitAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
No.  The TraitDiscoverer attribute does not allow multiple uses and it has to have differet values for the assemblyName parameter.  It's unfortunate.  I'd raelly like to not do this.